app/tests/api/inventory.jest.ts

```diff
@@ -306,7 +306,7 @@ describe("Inventory API", () => {
     const {
       error: { issues },
     } = await res.json();
-    expect(issues.length).toEqual(3);
+    expect(issues.length).toEqual(1);
```

Comment:
Why is it less issues? It seems something about our validation rules changed here?

Reply:
we have fewer issues because of:

export const upsertInventoryRequest = z.union([
  createInventoryRequest.strict(),
  z
    .object({
      isPublic: z.boolean().optional(),
    })
    .strict(),
]);

so zod is returning the smallest of errors, which is just the single "isPublic" field